RyanNoelk/OpenEats
# Feature/import recipes

Adding a way to import recipes into the site. For a full description of the problem see: https://github.com/RyanNoelk/OpenEats/issues/177

The only thing left to do is to add tests.

To Test: 
- docker-compose build 
- make sure that pip requirements are installed
- make sure node packages are installed
- There are only two sites supported right now. You should be able to view them using the interface.
- Import a recipe from one of those sites.
_ Make sure the recipe creation page has the correct data.
- Make sure the image is also created when the user clicks submit.

@hermzz let me know if you don't have time to test this out.
